aea/helpers/ipfs/base.py

```diff
@@ -43,35 +41,13 @@
     from aea.helpers.ipfs.pb.merkledag_pb2 import PBNode  # type: ignore
 
 
-def _dos2unix(file_content: bytes) -> bytes:
-    """
-    Replace occurrences of Windows line terminator CR/LF with only LF.
-
-    :param file_content: the content of the file.
-    :return: the same content but with the line terminator
-    """
-    return re.sub(b"\r\n", b"\n", file_content, flags=re.M)
-
-
-def _is_text(file_path: str) -> bool:
-    """Check if a file can be read as text or not."""
-    try:
-        with open_file(file_path, "r") as f:
-            f.read()
-        return True
-    except UnicodeDecodeError:
-        return False
-
-
 def _read(file_path: str) -> bytes:
     """Read a file, replacing Windows line endings if it is a text file."""
```

Comment:
@angrybayblade i dont see any fix for windows line endings anymore, file is opened in binary mode, so python will preserve all chars. 

probably doc string is needed to be updated?

Reply:
Fixed [fb89dfe](https://github.com/valory-xyz/open-aea/pull/674/commits/fb89dfea890f1e1a2afa51e83843a001c7610625)